specification/resources/resource-manager/readme.azureresourceschema.md

```diff
@@ -387,6 +387,7 @@ input-file:
   - Microsoft.Authorization/stable/2020-09-01/policyAssignments.json
   - Microsoft.Authorization/stable/2020-09-01/policyDefinitions.json
   - Microsoft.Authorization/stable/2020-09-01/policySetDefinitions.json
+  - Microsoft.Resources/stable/2020-06-01/resources.json
```

Comment:
Won't this also result in this tag generating schemas for Microsoft.Resources? It's obviously not ideal to have a cross-API-version reference, but shouldn't it be allowed without this change?

Reply:
oh you're right. With the change, it will generate Resources schemas too. I will try to find the cause in autorest.